load_fund_echo/routes.py

```diff
@@ -0,0 +1,103 @@
+import json
+from os import getenv
+from db import DatabaseManager
+from flask import request, jsonify, Blueprint
```

Comment:
We are data engineers and Python provides us a very powerful weapon called - Pandas Dataframe for doing data engineering.
So, make use of it because it is more readable and it reduces line of code.

List and tuple are index based data types. It does not guide us the actual content of the data structure.